src/libai/classifiers/bayes/NaiveBayes.java

```diff
@@ -113,9 +113,9 @@ private void precalculate(DataSet ds) {
 					double mean = acum.first;
 					double length = (double) ((Integer) data[outputIndex]);
 
-					sd = (sd - ((mean * mean) / (double) length));
-					acum.second = sd / (double) (length - 1);
-					acum.first = mean / (double) length;
+					sd = (sd - ((mean * mean) / length));
```

Comment:
Please roll back this change. I know it seems silly but there was something not working as expected without the castings. Until we have proper unit test for this classes, i rather keep the changes to the minimum.